### PR TITLE
Feature/droid 269 update to android 14

### PR DESCRIPTION
**Pull request by nick-sasquatch**

### CODE REVIEW

File: combustion-android-ble/src/main/java/inc/combustion/framework/service/CombustionService.kt [hunk 1]
```diff
@@ -113,7 +115,11 @@ class CombustionService : LifecycleService() {
 
     private fun startForeground() {
         serviceNotification?.let {
-            startForeground(notificationId, serviceNotification)
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
```
Comment (nick-sasquatch) [inline combustion-android-ble/src/main/java/inc/combustion/framework/service/CombustionService.kt:118]: This seems to be required as of Android 14 (https://developer.android.com/about/versions/14/changes/fgs-types-required).

File: combustion-android-ble/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    package="inc.combustion.framework">
```
Comment (angrygorilla) [inline combustion-android-ble/src/main/AndroidManifest.xml:4]: why did this need to be moved out? just out of curiosity.

**Review by angrygorilla — APPROVED** [review]
nice job!

**Comment by angrygorilla** [comment]
Tested out build with corresponding app PR on pixel 8 android 14. I will test on older pixel 5 with android 11 (can't remember phone is dead)

File: combustion-android-ble/src/main/AndroidManifest.xml (hunk 2 above)
Comment (nick-sasquatch) [reply combustion-android-ble/src/main/AndroidManifest.xml:4]: I think that was a miss--thanks for catching it!

**Pull request merged**